Servers/WindowServer/WSWindowManager.cpp

```diff
@@ -112,6 +112,28 @@ WSWindowManager::~WSWindowManager()
 {
 }
 
+Retained<WSCursor> WSWindowManager::get_cursor(const String& name, const Point& hotspot)
+{
+    auto path = m_wm_config->read_entry("Cursor", name, "/res/cursors/arrow.png");
+    auto gb = GraphicsBitmap::load_from_file(path);
+    if (gb) {
+        return WSCursor::create(*gb, hotspot);
+    } else {
+        return WSCursor::create(*GraphicsBitmap::load_from_file("/res/cursors/arrow.png"));
+    }
+}
+
+Retained<WSCursor> WSWindowManager::get_cursor(const String& name)
+{
+    auto path = m_wm_config->read_entry("Cursor", name, "/res/cursors/arrow.png");
+    auto gb = GraphicsBitmap::load_from_file(path);
+    if (gb) {
+        return WSCursor::create(*gb);
+    } else {
```

Comment:
No need to "else" after a return statement. We don't have to fix this now, but going forward, let's try to avoid nesting whenever possible :)